hbase-client/src/main/java/org/apache/hadoop/hbase/client/RpcRetryingCallerWithReadReplicas.java

```diff
@@ -95,8 +95,8 @@ class ReplicaRegionServerCallable extends CancellableRegionServerCallable<Result
     public ReplicaRegionServerCallable(int id, HRegionLocation location) {
       super(RpcRetryingCallerWithReadReplicas.this.cConnection,
         RpcRetryingCallerWithReadReplicas.this.tableName, get.getRow(),
-        rpcControllerFactory.newController(), rpcTimeout, new RetryingTimeTracker(),
-        PRIORITY_UNSET);
+        rpcControllerFactory.newController(), rpcTimeout, new RetryingTimeTracker(), PRIORITY_UNSET,
+        Collections.emptyMap());
```

Comment:
This IA.Private class is constructed by HTable for use in replica gets. I think you should add request attributes to the constructor so that they can be passed in here